<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWidget</class>
 <widget class="QWidget" name="SecondWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>端口：9999</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>对方IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 177, 177);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>对方端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Port">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SendMsg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
